pyccel/codegen/printing/ccode.py

```diff
@@ -209,7 +210,45 @@ def _print_PyccelNot(self, expr):
         return '!{}'.format(a)
 
     def _print_Import(self, expr):
-        return '#include "{0}"'.format(expr.source)
+        return '#include <{0}>'.format(expr.source)
+
+    def _print_String(self, expr):
+        return "\"{}\"".format(expr.arg)
+
+    def _print_Print(self, expr):
+        self._additional_imports.add("stdio.h")
+        args = []
+        args_format = []
+        end = '\\n'
+        sep = ' '
+        for f in expr.expr:
+            if isinstance(f, ValuedVariable):
+                if f.name == 'sep'      :   sep = str(f.value)
+                elif f.name == 'end'    :   end = str(f.value)
+            elif f.dtype is NativeBool():
+                args_format.append("%s")
+                args.append('({}) ? "True" : "False"'.format(self._print(f)))
+            elif f.dtype is NativeInteger():
+                args_format.append("%ld")
+                args.append(self._print(f))
+            elif f.dtype is NativeReal():
+                args_format.append("%lf")
+                args.append(self._print(f))
+            elif f.dtype is NativeString():
+                args_format.append("%s")
+                args.append(self._print(f))
+            elif f.dtype is NativeComplex():
+                args_format.append("(%f + %fj)")
```

Comment:
If you make the `if` block in this `for` loop into a function returning the arg_format and the arg then you could call the function recursively to set the format of the real and imaginary parts of the complex.